### ISSUE TITLE
`taup_pierce -pierce` of TauP v2.4.0 does not work

### ISSUE THREAD

**Issue by seisman** [crotwell/TauP#1]
`taup_pierce -pierce` of TauP v2.4.0 does not output the travel time at the depth specified by `-pierce` option.

Ouput from TauP v2.3.0:
``` bash
$ /opt/TauP-2.3.0/bin/taup_pierce -mod prem -h 600 -deg 45 -ph PKiKP -pierce 5049.5
> PKiKP at   944.86 seconds at    45.00 degrees for a    600.0 km deep source in the prem model with rayParam    1.023 s/deg.
    0.00   600.0     0.0
    0.07   670.0     6.9
    4.96  2891.0   188.1
   21.23  5049.5   427.9        <--- travel time at depth 5049.5
   22.26  5149.5   437.5
   23.29  5049.5   447.1        <--- travel time at depth 5049.5
   39.55  2891.0   686.9
   44.44   670.0   868.1
   44.52   600.0   875.0
   44.71   400.0   895.8
   44.85   220.0   916.5
   44.99    24.4   940.9
   44.99    15.0   942.3
   45.00     0.0   944.9
```

Output from TauP v2.4.0:
```
$ /opt/TauP-2.4.0/bin/taup_pierce -mod prem -h 600 -deg 45 -ph PKiKP -pierce 5049.5
> PKiKP at   944.83 seconds at    45.00 degrees for a    600.0 km deep source in the prem model with rayParam    0.973 s/deg.
    0.00   600.0     0.0
    0.07   670.0     6.9
    4.69  2891.0   187.8
   22.27  5149.5   437.5
   39.86  2891.0   687.2
   44.47   670.0   868.1
   44.54   600.0   875.0
   44.72   400.0   895.8
   44.86   220.0   916.5
   44.99    24.4   940.9
   44.99    15.0   942.3
   45.00     0.0   944.9
```

**Comment by crotwell** [comment]
Thanks for reporting this. Fixed in commit 
https://github.com/crotwell/TauP/commit/039112bf67104b3e29f4164f79465142f2b108fa

Fix will be in v 2.4.1, hopefully out shortly.

**Comment by crotwell** [comment]
See version 2.4.1
http://www.seis.sc.edu/taup/

thanks again for reporting this
Philip
